### PR TITLE
Refactor bq/simplify operators rename hooks

### PR DESCRIPTION
**Pull request by TobKed**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

File: tests/gcp/hooks/test_bigquery.py [hunk 1]
```diff
@@ -139,98 +140,50 @@ def test_throws_exception_with_incompatible_syntax(self, mock_project_id):
...
+
+    @parameterized.expand([
+        ("project", "dataset", "table", "dataset.table", "project"),
+        ("alternative", "dataset", "table", "alternative:dataset.table", "project"),
+        ("alternative", "dataset", "table", "alternative.dataset.table", "project"),
+        ("alt1:alt", "dataset", "table", "alt1:alt.dataset.table", "project"),
+        ("alt1:alt", "dataset", "table", "alt1:alt:dataset.table", "project"),
+    ])
+    def test_split_tablename(
+        self, project_expected, dataset_expected, table_expected, table_input, default_project_id
```
Comment (mik-laj) [inline tests/gcp/hooks/test_bigquery.py]: Is defaullt_project_id a constant or variable value?

File: tests/gcp/hooks/test_bigquery.py [hunk 2]
```diff
@@ -80,66 +72,6 @@ def test_credentials_provided(self, mock_read_gbq, _):
         self.assertEqual("PROJECT_ID", kwargs['project_id'])
 
 
-class TestBigQueryDataframeResults(unittest.TestCase):
```
Comment (mik-laj) [inline tests/gcp/hooks/test_bigquery.py]: Why did you delete it?

**Comment by mik-laj** [comment]
Can you also improve TestBigQueryHookSourceFormat,  TestBigQueryExternalTableSourceFormat? They should use assertRaisesRegex
Many test methods  stil use hook.BigQueryBaseCursor.__init__

File: tests/gcp/hooks/test_bigquery.py (hunk 2 above)
Comment (TobKed) [reply tests/gcp/hooks/test_bigquery.py]: It is moved to `tests/gcp/hooks/test_bigquery_system.py` due to the fact that they require BigQuery connection. I forgot to add them to commit, I've made fixup.

File: tests/gcp/hooks/test_bigquery.py (hunk 1 above)
Comment (TobKed) [reply tests/gcp/hooks/test_bigquery.py]: It could be a constant across all these tests. I will change it.
